articles/openstack/ostack-how-use-cli.md

```diff
@@ -123,6 +123,8 @@ After successful installation, you'll have to download the OpenStack RC file for
 
     ![Download OpenStack RC File button in Horizon dashboard](images/ostack-horizon-download-rc-file.png)
 
+* If your user is SSO enabled, please see this guide for modifying your RC file to allow it to work with your SSO enabled user [Guide](ostack-sso-rcfile.md)
```

Comment:
I've suggested a change to the file name of the new article, so just make sure that the file name you use here matches the actual file name of the article.